lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false; // replace null with your response
```

Comment:
this is true because because between lines 7 and 8 is still within the scope of deathstar